openqaoa/optimizers/result.py

```diff
@@ -54,9 +54,10 @@ class Result:
     """
 
     def __init__(
-        self, log: Type[Logger], method: Type[str], cost_hamiltonian: Type[Hamiltonian]
+        self, log: Type[Logger], method: Type[str], cost_hamiltonian: Type[Hamiltonian], backend: str
```

Comment:
You also need to change the docs of the class. But did you agree with @raulconchello on this? Is this change consistent with the overall refactor of the result object?

Reply:
Yes, we did that together

Reply:
What is it in your PR?